packages/components/src/index.tsx

```diff
@@ -170,4 +193,26 @@ const CourierComponents: React.FunctionComponent = () => {
     </CourierProvider>,
     courierRoot
   );
-})();
+
+  hasInit = true;
+};
+
+window.courier = {
+  init: initCourier,
+  on: (action: string, cb: () => void) => {
+    actions[action] = actions[action] ?? [];
+    actions[action].push(cb);
+  },
+};
+
+if (window.courierConfig.initOnLoad !== false) {
+  initCourier();
```

Comment:
@RoyalSix i could pass window.courierConfig to this function so it works goodlike